src/main/java/org/desarrolladorslp/technovation/controller/MessageController.java

```diff
@@ -29,6 +28,19 @@
         return new ResponseEntity<>(messages, HttpStatus.OK);
     }
 
+    @PutMapping("/{messageId}/read")
+    public ResponseEntity markMessageAsRead(@PathVariable UUID messageId){
```

Comment:
When you are not returning anything and the response status is 200 you can simply use a `void` method